stable/datadog/values.yaml

```
@@ -48,6 +48,7 @@ datadog:
   ## modifying the label type below:
   #
   securityContext: {}
+  #   type: MustRunAs
   #   seLinuxOptions:
   #     user: system_u
   #     role: system_r
```

If we explicitly set the default value here, it will simplify the templates by removing the need to test `nil` value and to put the default value in the template.

```suggestion
  securityContext:
    type: MustRunAs
    seLinuxOptions:
      user: system_u
      role: system_r
      type: spc_t
      level: s0
```